layers/+emacs/org/README.org

```diff
@@ -413,6 +414,14 @@ Set the variable =org-enable-valign= to =t=:
 
 [[https://github.com/casouri/valign#valignel][Known problem: Rendering large tables (≥100 lines) is laggy.]]
 
+** Org-appear support
+To install [[https://github.com/awth13/org-appear][org-appear]], set the =org-enable-appear-support= to =t=:
```

Comment:
Can we have a high-level description about what org-appear does here? In the end we could refer to the original repository for more details.